### DISCUSSION TITLE
Office_S_Tips/epuap/2019/09/19/Pismo_ogolne_epuap

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#15] [Komentarze]
# Office_S_Tips/epuap/2019/09/19/Pismo_ogolne_epuap

Czasem trudno jest wystartować z pismem do urzędu/instytucji na ePUAP. Warto skorzystać z https://www.gov.pl/web/gov/wyslij-pismo-ogolne

https://andrzejq.github.io/Office_S_Tips/epuap/2019/09/19/Pismo_ogolne_epuap.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
